Dockerfile

```
@@ -68,10 +68,10 @@ RUN apt-get install -y software-properties-common \
 ENV PYENV_ROOT=/usr/local/.pyenv \
   PATH="/usr/local/.pyenv/bin:$PATH"
 RUN git clone https://github.com/pyenv/pyenv.git /usr/local/.pyenv \
-  && cd /usr/local/.pyenv && git checkout v1.2.22 && cd - \
-  && pyenv install 3.9.1 \
+  && cd /usr/local/.pyenv && git checkout 9ee109b66148bc39a685926050b7b56cb4bb184b && cd - \
```

Looks like it's released now https://github.com/pyenv/pyenv/releases/tag/v1.2.23

```suggestion
  && cd /usr/local/.pyenv && git checkout v1.2.23 && cd - \
```

---

Ah I see now that it doesn't include all the new versions.

---

Yeah, 1.2.23 is missing 3.9.2 and 3.8.8